app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # trường name không được bỏ trống
```

Comment:
Remove **ALL** comments in this file. Make a note or something that out of your project to keep your knowledges